### PR TITLE
Adjust footer spacing in mobile view

### LINKED ISSUE #1782: Adjust Footer Spacing In Mobile View
The spacing allows for better "tap" on mobile web

### Current portrait view
pixelfed-mobile-portrait-current

### Suggested portrait view
pixelfed-mobile-portrait-suggested

### Current landscape view
pixelfed-mobile-landscape-current

### Suggested landscape view
pixelfed-mobile-landscape-suggested

What do you think @dansup ?

### PR DESCRIPTION
**Pull request by okpierre**
Adjust Footer Spacing In Mobile View https://github.com/pixelfed/pixelfed/issues/1782

Fix #1782

### CODE REVIEW

**Review by trwnh — DISMISSED** [review]
LGTM!

**Review by dansup — APPROVED** [review]

**Comment by dansup** [comment]
Looks good, will merge after #1781 is merged!

**Pull request merged**
